README.md

```diff
@@ -453,6 +453,73 @@ As you can see all the classes in a class hierarchy actually share oneclass vari
   end
 ```
 
+- Avoid explicit use of instance variables
+
+```ruby
+  # bad
+  class SomeClass
+    def initialize(foo)
+      @foo = foo
+    end
+  end
+  
+  def bar
+    @foo == "Foo"
+  end
+  
+  # good
+  class SomeClass
+    attr_reader :foo
+    
+    def initialize(foo)
+      @foo = foo
+    end
+    
+    def bar
+      foo == "Foo"
+    end
+  end
+  
+  # better
+  class SomeClass
+    def initialize(foo)
+      @foo = foo
+    end
+    
+    def bar
+      foo == "Foo"
+    end
+    
+    private
+    
+    attr_reader :foo
+    
+  end
+```
+
+- Avoid complex logic in the initialiser
```

Comment:
This might need a "where possible" clause? You can't _always_ push work out to be done lazily as the other method is called, sometimes it needs doing from the initialiser. (Of course, it can be pushed to the other method & then called from the initialiser rather than having a complicated initialiser.)